.github/workflows/release.yml

```
@@ -0,0 +1,63 @@
+on:
+  push:
+    tags:
+      - 'r*'
+
+name: Create Release
+
+jobs:
+  build:
+    name: Create Release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Generate Changelog
+        run: |
+          prev_tag=$(git for-each-ref --sort=creatordate --format '%(refname:lstrip=2)' refs/tags | grep ^r | tail -2 | head -1)
+          curr_tag=$(git for-each-ref --sort=creatordate --format '%(refname:lstrip=2)' refs/tags | grep ^r | tail -1)
+          echo "Previous tag: ${prev_tag}"
+          echo "Current tag: ${curr_tag}"
+          echo "#### New rules:" > changes.txt
+          git log --pretty=%B ${prev_tag}..${curr_tag} | grep -E '^\s*new: ' | sort | sed -e 's%^% - %' >> changes.txt
+          echo "#### Changed rules:" >> changes.txt
+          git log --pretty=%B ${prev_tag}..${curr_tag} | grep -E '^\s*change: ' | sort | sed -e 's%^% - %' >> changes.txt
+          echo "#### Rules with fixes:" >> changes.txt
+          git log --pretty=%B ${prev_tag}..${curr_tag} | grep -E '^\s*fix: ' | sort | sed -e 's%^% - %' >> changes.txt
+          git log --pretty=%B ${prev_tag}..${curr_tag} | grep -oP 'Merge PR #\d+ from \K(@\S+)' | sort -u > authors_raw.txt
+          git log --pretty=%B ${prev_tag}..${curr_tag} | grep -oP "Co-authored-by: \K.*(?= <)" | sort -u | sed -e 's%^%@%' >> authors_raw.txt
+          LC_ALL=en_US.UTF-8 sort -u authors_raw.txt | grep -v 'dependabot\[bot\]' > authors.txt
+          echo "### Changes in this release:" > changelog.txt
+          cat changes.txt >> changelog.txt
+          echo "" >> changelog.txt
+          echo "Thanks to $(perl -pe 's%\n%, %' authors.txt | sed 's%, $%%')" >> changelog.txt
+          echo "" >> changelog.txt
+          echo "" >> changelog.txt
+          echo "#### Which Sigma rule package should I use?" >> changelog.txt
+          echo "A detailed explanation can be found in the [Releases.md](Releases.md) file. If you are new to Sigma, we recommend starting with the \"Core\" ruleset." >> changelog.txt
```

I moved the markdown with some changes to the wiki section. https://github.com/SigmaHQ/sigma/wiki/Sigma-Release-Packages we'll discuss it offline

```suggestion
          echo "A detailed explanation can be found in the release [wiki](https://github.com/SigmaHQ/sigma/wiki/Sigma-Release-Packages) file. If you are new to Sigma, we recommend starting with the \`Core\` ruleset." >> changelog.txt
```